src/serverenvironment.cpp

```diff
@@ -1923,12 +1919,20 @@ u16 ServerEnvironment::addActiveObjectRaw(std::unique_ptr<ServerActiveObject> ob
 					MOD_REASON_ADD_ACTIVE_OBJECT_RAW);
 		} else {
 			v3s16 p = floatToInt(objectpos, BS);
-			errorstream<<"ServerEnvironment::addActiveObjectRaw(): "
-				<<"could not emerge block for storing id="<<object->getId()
-				<<" statically (pos="<<p<<")"<<std::endl;
+			errorstream << "ServerEnvironment::addActiveObjectRaw(): "
```

Comment:
As I said before, this branch cannot be reached, since ServerMap::emergeBlock defaults to creating a blank block and will never return null unless false is passed in its second argument.

Reply:
Moving the `emergeBlock` call into `l_add_entity` should satisfy it.

Reply:
There's one reason it could return null but that's currently not handled correctly.
`ServerMap::createSector` throws an exception if the position is out of bounds but it's not caught.

Reply:
So, as is, we can just remove the else branch, no? (Assuming the exception case is fixed.)

Reply:
as of 9da1354f3ae5bb62f9cf067c3ae38d87d956d603 the else branch is no longer dead code